src/Neo.CLI/config.json

```diff
@@ -30,8 +30,8 @@
   "ProtocolConfiguration": {
     "Network": 860833102,
     "AddressVersion": 53,
-    "MillisecondsPerBlock": 15000,
-    "MaxTransactionsPerBlock": 512,
+    "MillisecondsPerBlock": 3000,
```

Comment:
And this one is not ready yet, I think. We should change testnet only first.

Reply:
I see, but I thought about the release itself. But if you want we can change it.